Source/WebCore/rendering/style/ListStyleType.cpp

```diff
@@ -33,22 +33,22 @@ namespace WebCore {
 
 bool ListStyleType::isCircle() const
 {
-    return type == Type::Circle || (type == Type::CounterStyle && identifier == nameLiteral(CSSValueCircle));
+    return type == Type::Circle;
 }
 
 bool ListStyleType::isSquare() const
 {
-    return type == Type::Square || (type == Type::CounterStyle && identifier == nameLiteral(CSSValueSquare));
+    return type == Type::Square;
 }
 
 bool ListStyleType::isDisc() const
 {
-    return type == Type::Disc || (type == Type::CounterStyle && identifier == nameLiteral(CSSValueDisc));
+    return type == Type::Disc;
 }
 
 bool ListStyleType::isDisclosureClosed() const
 {
-    return type == Type::DisclosureClosed || (type == Type::CounterStyle && identifier == nameLiteral(CSSValueDisclosureClosed));
+    return type == Type::DisclosureClosed;
 }
```

Comment:
UA counter styles may still be represented via `(type == Type::CounterStyle && identifier == nameLiteral(CSSValueDisclosureClosed))` no? I don't see how the change above prevents that from happening

Reply:
You are right. However, the original idea behind this in this spec was hard-coding these specific values for [avoiding hash-lookup costs](https://github.com/w3c/csswg-drafts/issues/3584). I'm wondering if we should add these to the list of unsupported styles, so they are always represented via the hard-coded path, but keep the rules at counterStyles.css for enabling them to be extended.